src/linker/Linker/PInvokeInfo.cs

```diff
@@ -1,9 +1,10 @@
-﻿using System.Runtime.Serialization;
+﻿using System;
+using System.Runtime.Serialization;
 
 namespace Mono.Linker
 {
 	[DataContract]
-	public class PInvokeInfo
+	public class PInvokeInfo : IComparable
```

Comment:
Use IComparable<PInvokeInfo>

Reply:
I implemented `CompareTo` and it's being used for sorting the PInvokes list in the OutputStep.